### DISCUSSION TITLE
death location placeholder

### DISCUSSION THREAD

**Discussion by Eutherin** [Zrips/CMI#3313] [Feature Requests]
**Description of feature/improvement request:**
Command that only shows death location.

**Comment by CloudeLecaw** [comment]
Why this would be needed? a new command? just for showing death locations?
We allready have 50% of this... it justs needs a little thing like a placeholder maybe.

We allready have
**cmi.deathlocation _– Allows to see death location after death_**
If you died and have this permission then your deathkords are shown in chat.

It would be nice to have a CMI - Place holder - may called: [deathlocation]
With this placeholder we could create our own CustomAlias that is printing out the language string that is allready existing:
`deathlocation: '&eYou died at x:&6[x]&e, y:&6[y]&e, z:&6[z]&e in &6[world]'`
as often as we (or the player) want in a msg to the player ;)
CustomAliases can also have permisisons

so adding extra new command just for print out the deathlocation would not be needed :)

Well if u may like the idea - you may could rename the Topiclable to "death location placeholder" ? <3

**Comment by Eutherin** [comment]
That makes a lot of sense, I guess that would be a better thing to add!

**Comment by x1p** [comment]
Following with interest.

**Comment by CloudeLecaw** [comment]
i think this ticket might could get closed since there is a placeholder allready in cmi for this now since a while
%cmi_user_deathloc% – Players last death location

this can be used in a custom alias that msgs you this placeholder
